doc/src/development/testenv/theo-setup-vm-windows.md

```diff
@@ -31,21 +31,33 @@ You may also install the required software manually.
   ssh-keygen -t rsa -b 4096 -C "opendut-vm" -f ~/.ssh/id_rsa
   ```
 
+> **Hint**  
+> If you do not want to change your global Windows git configuration, in order to avoid any 'crlf' issues on other repositories, you may consider using the cloned opendut repo on the Windows host only for the vagrant VM setup part.  
+> For working with THEO, use the cloned opendut repository inside the Linux guest system instead (`/home/vagrant/opendut`).  
+> You may also consult with [.gitattributes](https://git-scm.com/docs/gitattributes)
+
 ## Setup virtual machine
 
-* Add the following environment variables to point vagrant to the vagrant file
+* Add the following environment variables to point vagrant to the vagrant file  
+  Git Bash:
     ```sh
     export OPENDUT_REPO_ROOT=$(git rev-parse --show-toplevel)
     export VAGRANT_DOTFILE_PATH=$OPENDUT_REPO_ROOT/.vagrant
     export VAGRANT_VAGRANTFILE=$OPENDUT_REPO_ROOT/.ci/docker/Vagrantfile
     ```
-* Set up the vagrant box (following commands were tested in git-bash)
+    PowerShell:
+    ```powershell
+    $env:OPENDUT_REPO_ROOT=$(git rev-parse --show-toplevel)
+    $env:VAGRANT_DOTFILE_PATH="$env:OPENDUT_REPO_ROOT/.vagrant"
+    $env:VAGRANT_VAGRANTFILE="$env:OPENDUT_REPO_ROOT/.ci/docker/Vagrantfile"
+    ```
+* Set up the vagrant box (following commands were tested in Git Bash and Powershell)
 ```sh
 vagrant up
 ```
 
 
-> **Info**
+> **Info**  
```

Comment:
There sneaked in some spaces.

Reply:
The 2 spaces were introduced to have a line break after "Info".